<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Present_objects</class>
 <widget class="QDialog" name="Present_objects">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>150</y>
     <width>391</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI Semilight</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Firma:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI Semilight</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Firma:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>590</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Zamknij</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>520</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Nastepny</string>
   </property>
  </widget>
  <widget class="QPushButton" name="previous">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>450</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Poprzedni</string>
   </property>
  </widget>
  <widget class="QLabel" name="first_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
    </font>
   </property>
   <property name="text">
    <string>Firma:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="first_head">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="second_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
    </font>
   </property>
   <property name="text">
    <string>Moc [W]:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="second_head">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
    </font>
   </property>
  </widget>
  <widget class="QTextBrowser" name="third_head">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
    </font>
   </property>
  </widget>
  <widget class="QTextBrowser" name="fourth_head">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>330</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
    </font>
   </property>
  </widget>
  <widget class="QTextBrowser" name="fifth_head">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>390</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
    </font>
   </property>
  </widget>
  <widget class="QTextBrowser" name="sixth_head">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>450</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
    </font>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Nirmala UI'; font-size:7.875pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="seventh_head">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>510</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="third_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
    </font>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
  </widget>
  <widget class="QLabel" name="fourth_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
    </font>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
  </widget>
  <widget class="QLabel" name="fifth_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
    </font>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
  </widget>
  <widget class="QLabel" name="sixth_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
    </font>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
  </widget>
  <widget class="QLabel" name="seventh_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
    </font>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Present_objects</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>745</x>
     <y>591</y>
    </hint>
    <hint type="destinationlabel">
     <x>616</x>
     <y>544</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>next</sender>
   <signal>clicked()</signal>
   <receiver>Present_objects</receiver>
   <slot>next()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>633</x>
     <y>573</y>
    </hint>
    <hint type="destinationlabel">
     <x>673</x>
     <y>507</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>previous</sender>
   <signal>clicked()</signal>
   <receiver>Present_objects</receiver>
   <slot>previous()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>494</x>
     <y>588</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>520</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>next()</slot>
  <slot>previous()</slot>
 </slots>
</ui>
